api/v1alpha1/baseboardmanagement_types.go

```diff
@@ -64,6 +64,10 @@ type Connection struct {
 	// +kubebuilder:validation:MinLength=1
 	Host string `json:"host"`
 
+	// Port is the IPMI port of the BaseboardManagement.
```

Comment:
nit: this could be HTTP/Redfish, can you make this wording generic? Can you state what a default value is, if any?

Reply:
I think default would be `623`. I can set a `kubebuilder:default`